input/number/reports.md

```diff
@@ -0,0 +1,28 @@
+## This page should be used for casual reporting of issues.
+
+For help on formatting, see: 
+- [How to put multiple lines in a Markdown table cell (multiline table)](https://alvinalexander.com/text/how-put-multiple-lines-markdown-table-cell-multiline-table)
+- [Markdown table generator](https://www.tablesgenerator.com/markdown_tables)
+
+### <a name="valid-issues">Valid issue types:</a> 
+- Inconsistency with other similar input elements
+- Non-extensibly (functionality)
+- Non-extensibly (styling)
+- Lacks built-in accessibility 
+
+### Structure of a report
+
+#### Please use a <a href="#valid-issues">valid issue type</a>.
+
+##### At least one link is mandatory. 
+
+|  An example report                                                                                                                    |
+|---------------------------------------------------------------------------------------------------------------------|
+| <p><strong>Issue</strong>: `<valid issue type>`<br><strong>Links</strong>:<br>- `<link1>`<br>- `<link2>`<br>- `<link3>`<br><strong>Description</strong>:`<some description>`</p> |
```

Comment:
I'd remove all of this, if we feel a specific structure is desired (which I'm thinking we will) then we should spin up a template file that folks copy and start from

Reply:
Fair, enough, but what would you like in the meantime?